staging/src/k8s.io/api/flowcontrol/v1alpha1/types.go

```diff
@@ -438,6 +438,30 @@ type LimitedPriorityLevelConfiguration struct {
 
 	// `limitResponse` indicates what to do with requests that can not be executed right now
 	LimitResponse LimitResponse `json:"limitResponse,omitempty" protobuf:"bytes,2,opt,name=limitResponse"`
+
```

Comment:
@MikeSpreitzer I copied it from the KEPs, can you please review these two fields? (I am working on fixing the other errors)